### PR TITLE
W-10920711: Add new steps for configuring tags and categories

### PR DESCRIPTION
**Pull request by fermujica**
[W-10920711](https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07EE00000tw30MYAQ/view): Based on new comprehensive publication to Exchange, the Publish Asset to Exchange task must be updated to include tags and categories configurations.

### CODE REVIEW

File: modules/ROOT/pages/export-to-exchange-task.adoc [hunk 1]
```diff
@@ -7,19 +7,29 @@ Anypoint Studio allows you to connect to Exchange to publish your project as a n
 
 . Right-click your project folder in your Package Explorer view.
 . Select *Anypoint Platform* > *Publish to Exchange*.
++
+image::publish-project-to-exchange.png[alt=Right-click on project and selecting Anypoint Platform and Publish to Exchange.]
```
Comment (Crispy-Salesforce) [inline modules/ROOT/pages/export-to-exchange-task.adoc]: Coherence:
Right-click + select + publish (recommended)
Right-clicking + selecting + publishing

File: modules/ROOT/pages/export-to-exchange-task.adoc [hunk 2]
```diff
@@ -7,19 +7,29 @@ Anypoint Studio allows you to connect to Exchange to publish your project as a n
...
 . Either select your Anypoint Platform user and business group or create a new Anypoint Platform account by clicking *Add Account*. +
 Additionally you can click the "Add Account" button to use a new Anypoint Platform account.
```
Comment (Crispy-Salesforce) [inline modules/ROOT/pages/export-to-exchange-task.adoc]: Comma missing:
Additionally, ...

File: modules/ROOT/pages/export-to-exchange-task.adoc [hunk 3]
```diff
@@ -7,19 +7,29 @@ Anypoint Studio allows you to connect to Exchange to publish your project as a n
...
 The group ID for your project is the organization ID of your Anypoint Platform organization.
-* *Artifact ID* +
+<1> *Artifact ID* +
 The artifact ID is the name of your project, minus its version. By default, Studio uses the artifact ID configured in your `pom.xml` file.
-* *Version*: The version of the asset you are publishing.
-. Select a project type. +
+<1> *Version*: The version of the asset you are publishing.
+<1> *Project type*: +
 If you are publishing a new version of an already published asset, you cannot change the project type.
 . Select *Save updated project information to the pom.xml*  if you want to update your `pom.xml` file with the data you just provided. +
```
Comment (Crispy-Salesforce) [inline modules/ROOT/pages/export-to-exchange-task.adoc]: There seems to be a double space between "...pom.xml*" and "if you want to...".

File: modules/ROOT/pages/export-to-exchange-task.adoc [hunk 4]
```diff
@@ -7,19 +7,29 @@ Anypoint Studio allows you to connect to Exchange to publish your project as a n
...
-This option updates your local project information with the data configured in the *Publish to Exchange* window. If you do not select this option, the project is deployed with the configured data, but your local project information is not updated.
-. Select *Finish*. +
+This option updates your local project information with the data you just configured. If you do not select this option, the project is deployed with the configured data, but your local project information is not updated.
+. Select *Next*.
+. Configure the tags and categories for your project:
++
+image::tags-and-categories-configuration.png[alt=configuration screen for adding tags and categories to your asset.]
```
Comment (Crispy-Salesforce) [inline modules/ROOT/pages/export-to-exchange-task.adoc]: According to our accessibility guidelines (https://confluence.internal.salesforce.com/display/MTDT/Writing+for+Accessibility), alt text should follow grammar rules. So, the first word of an alt text should be capitalized ("Configuration...").

File: modules/ROOT/pages/export-to-exchange-task.adoc [hunk 5]
```diff
@@ -7,19 +7,29 @@ Anypoint Studio allows you to connect to Exchange to publish your project as a n
...
+. Select *Finish*.
 
 After you select *Finish*, the publication of your project as a new asset in Exchange begins and cannot be cancelled. You can follow the progress of this publication process in the *Console* view.
```
Comment (Crispy-Salesforce) [inline modules/ROOT/pages/export-to-exchange-task.adoc]: The antecedent "publication" is a little too far from the verb phrase "cannot be cancelled", and there is a passive voice.
What about rewording it like?: 
Once you select *Finish*, the publication of your project as a new asset in Exchange begins. You can follow the progress of this publication process in the *Console* view, but you cannot cancel it.

File: modules/ROOT/pages/export-to-exchange-task.adoc [hunk 6]
```diff
@@ -7,22 +7,31 @@ Anypoint Studio allows you to connect to Exchange to publish your project as a n
 
 . Right-click your project folder in your Package Explorer view.
 . Select *Anypoint Platform* > *Publish to Exchange*.
++
+image::publish-project-to-exchange.png[alt=Right-click + select + publish.]
 . Either select your Anypoint Platform user and business group or create a new Anypoint Platform account by clicking *Add Account*. +
-Additionally you can click the "Add Account" button to use a new Anypoint Platform account.
+Additionally, you can click the *Add *ccount8 button to use a new Anypoint Platform account.
```
Comment (Cristian-Venticinque) [inline modules/ROOT/pages/export-to-exchange-task.adoc]: typo: *ccount8

**Review by Cristian-Venticinque — APPROVED** [review]

**Pull request merged**